work_queue/src/bindings/python3/PyTask/README.md

```diff
@@ -0,0 +1,32 @@
+# PyTask
```

Comment:
This is a good introductory text, but please integrate it into the WQ manual (see docs/manuals/workqueue) so that others will be able to find it easily.